brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-3254 Fix: Color picker default icon changed after WP 6.3 release.

### Description

- https://brainstormforce.atlassian.net/browse/SPCTR-3254

### Screenshots

- https://www.awesomescreenshot.com/video/19887738?key=19e7604a8056d27e61922a54f1ce43e6

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Drag an drop any block.
- Go to Style
- Open any panel that has color setting like Background, Text, Border, Box shadow etc
- Make sure default Icon should be transparent and when Theme color is selected earth icon should come.

### Checklist:
